test/controllers/users/registrations_controller_test.rb

```diff
@@ -203,7 +203,6 @@ class Users::RegistrationsControllerTest < ActionDispatch::IntegrationTest
       patch user_registration_url, params: payload
     end
 
-    assert_select "div#error_explanation", { count: 0 }
     refute @controller.view_assigns['user'].errors.present?
```

Comment:
@ajaystha42 why are these assertions removed?

Reply:
becase We are rendering the js here